File: 0070-climbing-stairs/0070-climbing-stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
        int prev = 1;
        int prev2 = 0;
        int current = 0;
        for(int i=0; i<n; i++){
            current = prev + prev2;
            prev2 = prev;
            prev = current;
        }
        return prev;  
    }
};

File: 0070-climbing-stairs/0070-climbing-stairs.rs
impl Solution {
    pub fn climb_stairs(n: i32) -> i32 {
    let mut prev = 1;
    let mut prev2 = 0;
    let mut current = 0;
    for _ in 0..n {
        current = prev + prev2;
        prev2 = prev;
        prev = current;
    }
    prev
    }
}